### ISSUE TITLE
Makefile: Install fails to install libstdbuf.so due to globbing

### ISSUE THREAD

**Issue by LeonardKoenig** [uutils/coreutils#1202]
The command
```make
install:
        # [...]
        $(foreach lib, $(LIBS), $(INSTALL) $(BUILDDIR)/build/*/out/$(lib) $(INSTALLDIR_LIB)/$(PROG_PREFIX)$(lib) &&) :
        # [...]
```
fails when there are multiple dirs in `build/` that contain the `libstdbuf.so` (or any other library that would be copied, in theory). I still had `target/release/build/stdbuf-e3408bc0e9238b7c/out/libstdbuf.so` left over, next to `target/release/build/stdbuf-1269745ba58d8635/out/libstdbuf.so`, so the command expanded to
```
$ install $BUILDDIR/build/stdbuf-1269745ba58d8635/out/libstdbuf.so $BUILDDIR/build/stdbuf-e3408bc0e9238b7c/out/libstdbuf.so $PREFIX/lib/libstdbuf.so
```
which means that instead of the intended syntax
```
install [OPTION]... [-T] SOURCE DEST
```
the syntax
```
install [OPTION]... SOURCE... DIRECTORY
```
is used, thus failing with
```
install: target '$PREFIX/lib/libstdbuf.so' is not a directory
make: *** [Makefile:314: install] Error 1
```

There's also a `liblibstdbuf.so` produced directly in `target/release/` which is identical to the one we intend to copy, except of the filename being wrong.

**Comment by Arcterus** [comment]
Thanks for filing an issue.  This just seems to be because the Makefile is out-of-date, as there is no longer a need to copy a `libstdbuf.so` anywhere (it's automatically generated upon running `stdbuf`).  I'll try to fix this in the next couple of days.

**Comment by Arcterus** [comment]
So, I take that back.  It will take less than a couple days because I've added a fix to #1201.

**Comment by Arcterus** [comment]
This should be fixed now.  Let me know if something is still failing for you.
